compute/ref_arch/security_data/log_files.adoc

```diff
@@ -2,9 +2,7 @@
 
 Prisma Cloud writes to RFC compliant syslog, this syslog data can be
 injected by any number of time series data solutions as well as common
-SIEM solutions. Twitlock offers a few layers of detail in our syslog
+SIEM solutions. Prisma cloud offers a few layers of detail in our syslog
 messages which you can select to increase the verbosity of the messages.
-Prisma Cloud recommends leaving the messages at their defaults Prisma Cloud
-can also write to STDOUT as well as feed in to Prometheus. For more
-information see our 
-https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-guide-compute/audit/logging.html[logging documentation].
+Prisma Cloud recommends leaving the messages at their defaults and it can also write to STDOUT as well as feed into Prometheus. For more
+information see our https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-01/prisma-cloud-compute-edition-admin/audit/logging[logging documentation].
```

Comment:
Hi Divya, the links need to be context sensitive for on-prem vs SaaS.

This doc is very outdated and I plan to unpublish it from the doc site.